src/components/ScheduleGenerator/SchedulerPanelHeader.vue

```diff
@@ -0,0 +1,235 @@
+
+          As such, if we want to enforce the credit limit here, would have to implement some sort of reactive JavaScript listener.
+        -->
+        <input type="number" placeholder='"18"' min="0" max="30" class="credit-limit-input" />
+      </div>
+      <button class="add-requirement-button">+ Requirement</button>
+    </div>
+    <hr />
+    <!-- The Figma wrote the below as 'No Requirements added' but I believe that was a capitalization mistake. -->
+    <p class="no-requirements-added">No requirements added.</p>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  props: {
+    // current semester being generated for
+    selectedSemester: { type: String, required: true },
+    // (linked with requirements) whether any requirements have been added
+    generateScheduleButtonDisabled: { type: Boolean, required: true },
+  },
+});
+</script>
+
+<style scoped lang="scss">
+@import '@/assets/scss/_variables.scss';
+
+/*
+  NOTE: There is a fair amount of overlap between the styles here and the styles in
+  @/components/Requirements/RequirementHeader.vue.
+*/
+
+.top {
+  margin: 1.5rem 0 1rem 0;
+
+  &-small {
+    margin: 0px;
+  }
+}
+
+.header-title {
+  color: $darkGray;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: normal;
+  text-transform: capitalize;
+}
+
+.semester {
+  display: flex;
+  padding-bottom: 12.5px;
+
+  &-title {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    color: $lightPlaceholderGray;
+    padding-bottom: 6px;
+
+    &-button {
+      border: none;
+      background: none;
+      text-align: center;
+      display: flex;
+      align-items: center;
+
+      p {
+        flex-direction: column;
+      }
+    }
+
+    &-top {
+      text-align: center;
+      font-style: normal;
+      font-weight: normal;
+      font-size: 14px;
+      margin: 0;
+    }
+
+    &-bottom {
+      text-align: center;
+      font-style: normal;
+      font-weight: normal;
+      font-size: 12px;
+      line-height: 15px;
+    }
+  }
+
+  &:hover {
+    background: rgba(255, 255, 255, 0.15);
+  }
+}
+
+.semester-title-button {
+  border-bottom: 2px solid $emGreen;
+}
+
+.semester-title-top {
+  font-weight: 500;
+  color: $emGreen;
+}
+
+/* This is the actual text warning message. */
+.no-requirements-added {
+  color: #cecece;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: normal;
+  text-align: center;
+  margin-top: 30px;
+}
+
+.generate-schedule-button {
+  display: flex;
+  width: 100%;
+  height: 34px;
+  padding: 5px 32px;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+  flex-shrink: 0;
+  border-radius: 3px;
+  color: $white;
+  margin-bottom: 12px;
+  border: 1px solid $emGreen;
+  background: $emGreen;
```

Comment:
OK @nidhi-mylavarapu so by default it is green, however that is the color in the Figma so I think it should be ok.